internal/commands/push/command_test.go

```diff
@@ -807,9 +799,9 @@ Successfully pushed app up: eggcorn-abcde
 			assert.Equal(t, `Determining changes
 Creating draft
 Pushing changes
+Installed dependencies
 Deploying draft
 Deployment complete
```

Comment:
Do we already have a test that confirms that the dependencies were properly deployed? In this test we just mock. the `ImportDependenciesFn` function.

Reply:
I believe this test verifies that the dependencies are deployed:
https://github.com/10gen/realm-cli/blob/9718bafcc70f855c0e8638d653e546caf4b15b40/internal/cloud/realm/dependencies_test.go#L29